server/server_test.go

```diff
@@ -605,15 +620,18 @@ func (cli *testServerClient) runTestLoadDataForListPartition(t *testing.T) {
 		dbt.MustExec("delete from t")
 		cli.prepareLoadDataFile(t, path, "1 a", "3 c", "4 e")
 		dbt.MustExec(fmt.Sprintf("load data local infile %q into table t", path))
+		rows.Close()
 		rows = dbt.MustQuery("select * from t order by id")
 		cli.checkRows(t, rows, "1 a", "3 c", "4 e")
 		// Test load data meet duplicate error.
 		cli.prepareLoadDataFile(t, path, "1 x", "2 b", "2 x", "7 a")
 		dbt.MustExec(fmt.Sprintf("load data local infile %q into table t", path))
+		rows.Close()
```

Comment:
I think this can be moved to line 626? Right after `cli.checkRows(t, rows, "1 a", "3 c", "4 e")`

Reply:
Done